index.js

```diff
@@ -79,15 +82,13 @@ async function pay (plugin, {
     })
 
     const payStream = ilpConn.createStream()
+    await payStream.sendTotal(sendAmount)
```

Comment:
should be able to pass a custom timeout here just like in pull

Reply:
I'll create a different PR to update `pay`.

Reply:
#7